assets/src/components/mapPage/searchResultsByProperty.tsx

```diff
@@ -74,17 +92,85 @@ const SearchResultSection = ({
   )
 }
 
+const LocationSearchResultSection = ({
+  text,
+  limit,
+  selectLocation,
+  showMore,
+}: {
+  text: string
+  limit: number
+  selectLocation: (location: LocationSearchResult) => void
+  showMore: () => void
+}) => {
+  const locationSearchResults = useLocationSearchResults(text)
+
+  return (
+    <section
+      className="c-map-page__search_results_section"
+      aria-labelledby={`search-results__location`}
+    >
+      <h2
+        className="c-map-page__search_results_header"
+        id={`search-results__location`}
+      >
+        {searchPropertyDisplayConfig.location.name}
+      </h2>
+      {locationSearchResults === null ? (
+        <Loading />
+      ) : locationSearchResults.length > 0 ? (
+        <>
+          <ul className="c-search-results__list">
+            {locationSearchResults
+              .slice(0, limit)
+              .map((locationSearchResult) => (
+                <li key={locationSearchResult.id}>
+                  <Card
```

Comment:
suggestion (non-blocking): This may already be on your radar for polish, but maybe it would be worth breaking this location card out into it's own component

Reply:
Yes, especially as I add things like the icon I suspect that it will make sense to split it out into a separate component, but I think that makes more sense to do in the followup PR once I have a better idea of how those changes are shaking out.